docs/CHANGELOG.md

```diff
@@ -0,0 +1,7244 @@
+    `SilKit/include/silkit/services/orchestration/SyncDatatypes.hpp` to
+    `OrchestrationDatatypes.hpp`.
+-   Participants may not be coordinated and not part of the required
+    participants list
+    -   Currently, this will lead to an exception
+-   Lifecycle service changes
+    -   Instead of booleans, the
+        `Service::Orchestration::LifecycleConfiguration` now comprises a
+        single enumerator `OperationMode` that defines if a participant
+        coordinates its state transition with others or if it runs
+        autonomously.
+    -   Most SystemCommands and all ParticipantCommands were removed.
+    -   Participants will not wait for a commands to Initialize, Run,
+        Stop, or Shutdown anymore. Instead, coordinated participants
+        will react to system state changes.
+    -   Instead of calling [ISystemController::Stop()]{.title-ref}, any
+        required participant can stop all coordinated participants by
+        calling [ILifecycleService::Stop()]{.title-ref}.
+    -   Autonomous participants must call
+        [ILifeCycleService::Stop()]{.title-ref} by themselves.
+    -   All participants that arrive at the `Stopped` state now continue
+        to `Shutdown` (via `ShuttingDown`)
+    -   The `Service::Orchestration::LifecycleConfiguration` must now be
+        provided in [IParticipant::CreateLifecycleService()]{.title-ref}
+        instead of [ILifecycleService::StartLifecycle()]{.title-ref}
+
+### Added
+
+-   The C API now has methods to aquire SIL Kit version information in
+    `SilKit/include/silkit/capi/Version.h`:
+
+    ``` {.sourceCode .c++}
+    SilKitAPI SilKit_ReturnCode SilKit_Version_Major(uint32_t* outVersionMajor);
+    SilKitAPI SilKit_ReturnCode SilKit_Version_Minor(uint32_t* outVersionMinor);
+    SilKitAPI SilKit_ReturnCode SilKit_Version_Patch(uint32_t* outVersionPatch);
+    SilKitAPI SilKit_ReturnCode SilKit_Version_BuildNumber(uint32_t* outVersionBuildNumber);
+    SilKitAPI SilKit_ReturnCode SilKit_Version_String(const char** outVersionString);
+    SilKitAPI SilKit_ReturnCode SilKit_Version_VersionSuffix(const char** outVersionVersionSuffix);
+    SilKitAPI SilKit_ReturnCode SilKit_Version_GitHash(const char** outVersionGitHash);
+    ```
+
+### Removed
+
+-   `SilKit/include/silkit/services/ethernet/EthernetDatatypes.hpp`:
+    Removed field `MacAdress` from `EthernetFrameTransmitEvent`.
+
+\[3.99.29\] - 28-07-2022
+------------------------
+
+### Compatibility with 3.99.28
+
+-   Application binary interface (ABI): No
+-   Application software interface (API): No
+-   Middleware network protocol: Yes
+
+### Added
+
+-   Added a
+    :cpp`SetCommunicationReadyHandlerAsync<SilKit::Services::Orchestration::ILifecycleServiceNoTime::SetCommunicationReadyHandlerAsync>`{.interpreted-text
+    role="func"} method to the lifecycle interfaces. It will invoke the
+    :cpp`CommunicationReadyHandler<SilKit::Services::Orchestration::CommunicationReadyHandler>`{.interpreted-text
```

Comment:
the breathe expansion of the doxygen references doesn't seem to work in the rendered HTML either